src/Gateways/Wechat.php

```diff
@@ -163,7 +163,7 @@ public function pay($gateway, $params = [])
      * @throws InvalidSignException
      * @throws \Yansongda\Pay\Exceptions\InvalidArgumentException
      *
-     * @return Collection
+     * @return WeChatPayRefundCollection|WeChatOfficialAccountNoticeCollection
```

Comment:
```suggestion
     * @return Collection
```

感谢支持！

这个问题已经在 supports 项目中有说明。

麻烦去掉后 merge

Reply:
已经移除。